### ISSUE TITLE
[Crash]: When joining fresh server/Random Disconnects

### ISSUE THREAD

**Issue by OneMoreStimpak** [AllTheMods/ATM-9#1885]
### Possible Fixes

Yes

### Modpack Version

0.2.61

### What happened?

When joining a fresh server I have started on a dedicated server, I am getting disconnected randomly. Some players are not able to join, being kicked immediately, while I am sometimes able to be online for a bit before it happens. I have been trying loads of things, java arguments, versions of OpenJava, Ram Amounts, Firewall Disable/rules. I am honestly just at a lost on what to do.

I have a shared hosting plan with bisect and everything works fine for them, so I know its something I am doing wrong. Just not sure what else to try. I don't understand the errors, and it points out a couple mods. Those mods work fine on bisect however so im just not sure. Any help is appreciated.   

### Crash Log

https://gist.github.com/OneMoreStimpak/f7eedb35c00494959d613708bad27e0f

### Latest.Log

https://gist.github.com/OneMoreStimpak/6b37d33b0a02aa67d9b1430449e5401c

**Comment by K-Karol** [comment]
I've just setup ATM9 today (Version 0.2.61) on a Pebblehost premium server I've been using with little to no issues with an FTB modpack, but with ATM it is also crashing for us with the same error:
 ```
 java.lang.NullPointerException: Cannot invoke "net.minecraft.client.multiplayer.ClientPacketListener.m_104910_()" because the return value of "net.minecraft.client.Minecraft.m_91403_()" is null
 ```
 
 What is odd is the server stops with no reason given, just stops:
 
 ```
 06.07 13:57:56 [PebbleHost Loader] Server Stopped
06.07 13:57:56 [PebbleHost Loader] PebbleHost Server Loader Stopping...
06.07 13:57:56 [PebbleHost Loader] Server loader version: v3.0.5 (using configuration version v1.3.8)
06.07 13:57:56 [PebbleHost Loader] PebbleHost Server Loader Stopped.
06.07 13:57:56 [PebbleHost] Server shut down (running)
06.07 13:57:56 [PebbleHost] Server stopped
06.07 13:57:56 [Disconnect] User <> has disconnected, reason: Server shutting down
```

**Comment by OneMoreStimpak** [comment]
Yeah I banged my head against the "wall," for 3 days trying to figure this out. Last night I finally decided to switch hosts from one company to one I have used before and liked. Now I have zero issues, unfortunately this doesn't really help anyone who might run into this but I simply couldn't figure out a fix. 

My solution was switching dedicated server hosts.
